js/myProfile.js

```diff
@@ -31,6 +36,18 @@ async function removePostFromTable(userID, productID){
     } else {
         newProductSellingList = userSellingProductIDList
     }
-    const respDynamoAddProductToUser = await docClientDynamo.updateTableEntry('UserInformation', userID, 'ListofProductIDSelling', newProductSellingList);
-    // error check? 
+    const respDynamoRemoveProductToUserSelling = await docClientDynamo.updateTableEntry('UserInformation', userID, 'ListofProductIDSelling', newProductSellingList);
+    const userSoldProductIDList = arrayAppend(respDyanmoGetUserEntry.Item['ListofProductIDSold'], productID);
```

Comment:
It looks like "arrayAppend" is not defined. Go to createPost.js and copy that function to this file.